tigerpath/majors_and_certificates/majors/CEE_2018.json.txt

```diff
@@ -0,0 +1,846 @@
+{
+    "type": "Major",
+    "name": "Civil and Environmental Engineering",
+    "code": "CEE",
+    "year": 2018,
+    "description": "The department offers five sub-plan options: architecture and engineering, environmental engineering, geological engineering, structural engineering, and engineering and the liberal arts. In the freshman year, students should complete the mathematics basic science, computing, and writing requirements common to all B.S.E. programs. At the end of the freshman year, the student declares CEE as a major and selects one of the subplans. Course requirements include engineering science courses, engineering design courses, a senior thesis, and program electives. For all but the non-ABET accredited tracks, there are additional math and science requirements that are normally satisfied by taking MAE 305 Mathematics in Engineering, ORF 245 Fundamentals of Engineering Statistics and GEO 203 Fundamentals of Solid Earth Science. In total, a student's program of study must include a minimum of fourteen engineering courses, with the exception of students in the non-ABET accredited tracks.",
+    "urls": [
+        "https://ua.princeton.edu/academic-units/department-civil-and-environmental-engineering#"
+    ],
+    "allowed_majors": [],
+    "contacts": [
+        {
+            "type": "Departmental Representative",
+            "name": "Branko Gli�ic",

```

Comment:
Do we support characters with diacritics? How well do the display in the frontend?